### PR TITLE
Added takeover

### PR DESCRIPTION
**Pull request by AndreasArledal**
screencast 2015-11-20 14-52-07

### CODE REVIEW

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 1]
```diff
@@ -0,0 +1,145 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  }
+}
+
+.takeover-body {
+  @extend .modal-body;
+  @extend %centered-content;
+
+  p:last-child {
+    margin-bottom: 0;
+  }
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: Rule declaration should be followed by an empty line

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+}
+
+.takeover {
+  @extend .modal;
+}
+
+.takeover-dialog {
+  @extend .modal-dialog;
+
+  background-color: rgba(255, 255, 255, .96);
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: `.96` should be written with a leading zero as `0.96`

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    </div>
+  </div>
+</div>
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#wide-takeover">
+  Launch wide takeover
+</button>
+```
+*/
+%centered-content {
+  max-width: 550px;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: Properties should be ordered margin, max-width

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+}
+
+.takeover-dialog {
+  @extend .modal-dialog;
+
+  background-color: rgba(255, 255, 255, .96);
+  bottom: 0;
+  left: 0;
+  margin: 0;
+  position: fixed;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: Properties should be ordered background-color, bottom, left, margin, overflow, position, right, top, width

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+  top: 0;
+  width: auto;
+  overflow: auto;
+
+}
+// h1 56
+// preamble 18
+.takeover-content {
+  @extend .modal-content;
+  border: none;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: Properties should be ordered background-color, border, border-radius, box-shadow, margin, overflow, position

File: src/mnd-bootstrap/javascript/_takeover.scss [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: auto;
+}
+
+.takeover-content {
+  @extend .modal-content;
+  background-color: transparent;
+  border-radius: 0;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/javascript/_takeover.scss]: Properties should be ordered background-color, border, border-radius, box-shadow, margin, overflow, position

File: src/mnd-bootstrap/javascript/_takeover.scss (hunk 2 above)
Comment (AndreasArledal) [inline src/mnd-bootstrap/javascript/_takeover.scss]: This one is actually fixed, seems like git doesn't recognize that it's an outdated diff.

**Comment by alexwoll** [comment]
:+1: like it

**Pull request merged**